charts/karpenter/templates/post-install-hook.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: helm-{{ .Release.Name }}-post-install-hook
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "karpenter.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade,post-rollback
+    "helm.sh/hook-delete-policy": hook-succeeded
+    "helm.sh/hook-delete-policy": hook-failed
+    {{- with .Values.additionalAnnotations }}
+      {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  ttlSecondsAfterFinished: 0
+  template:
+    spec:
+      serviceAccountName: {{ include "karpenter.serviceAccountName" . }}
+      restartPolicy: OnFailure
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: post-install-job
+        image: public.ecr.aws/bitnami/kubectl:1.30
+        command:
+        - /bin/sh
+        - -c
+        - |
+          if  "{{ .Values.webhook.enabled }}" == "true"; then 
```

Comment:
Is there not a way to selectively include the first part or the second part rather than resolving the value and using that in a bash script? Seems like you can do all the other helm logic up above.